code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -661,7 +661,7 @@
 
 /obj/item/reagent_containers/food/drinks/bottle/small/hellshenpa
 	name = "Hellshen Pale Ale"
-	desc = "The best ale on Mars, according to the label."
+	desc = "The best ale on Mars... According to the label- allegedly."
```

Comment:
Bruh

Reply:
Bruh.

Reply:
desc = "According to the label, this is the best Ale on mars."